source/frontend/src/assets/scss/App.scss

```diff
@@ -1,19 +1,112 @@
 @import './styles.scss';
 
-.modal-dialog.modal-xl {
-  max-width: 100rem;
-}
+.modal-dialog {
+  div.modal-header {
+    height: 48px;
```

Comment:
all of our sizing should be done in rems if possible.

so 4.8rem
0 1.6rem
etc

Reply:
Updated